### DISCUSSION TITLE
Copier wrapper

### DISCUSSION THREAD

**Discussion by martynvdijke** [copier-org/copier#1653] [Show and tell]
Hey maintainers,

In my off time I made a custom copier-wrapper written in Python.
Which allows feeding in a .CSV file containing all repositories it should update, to update to the latest git release.

Would you guys recommend adding this functionality in copier, or should I open source it my self ?

**Comment by pawamoy** [comment]
We can't answer without seeing the code ;)

More seriously, yes, you *could* open source it. Not saying you *should*, because open source is time consuming, so it's completely OK to keep it private.

We probably won't integrate this in Copier itself though, I think it better lives as a wrapper :+1:
